javascript/javascript3/week3/screenshot.js

```diff
@@ -0,0 +1,19 @@
+// ---- function to fetch the screenshot api ----
+
+const getScreenShot = async (link) => {
+  const url = `https://website-screenshot6.p.rapidapi.com/screenshot?url=${link}&width=1920&height=1080`;
```

Comment:
This might have worked this time, on your screen but may not look proper on all screens since the aspect ratio of the image itself will be different, it is not just about how it is displayed.

Reply:
Alright, I understand you . going to investigate this more :hugging_face: